### PR TITLE
Add google form to page, add summary, and formatting

### PR DESCRIPTION
**Pull request by belcherj**

### CODE REVIEW

**Comment by hectcastro** [comment]
Looks good. :+1:

**Comment by ocxo** [comment]
:thumbsup:

File: app/index.html [hunk 1]
```diff
@@ -2,41 +2,106 @@
 layout: default
 title: PhillyDev
 ---
-<h2>Want to sign up?</h2>
+<h2>What is PhillyDev</h2>
 
-<a target="_blank" href="https://docs.google.com/forms/d/1DomPnLHCvNksGlvO2QzOKlW-UHto6iFNAntjUkzKW20/viewform">Click here</a>
+<p>
+  What is PhillyDev?  PhillyDev is a place for developers who live and work in
+  and work around Philly to share, converse, organize and get to know each
```
Comment (chadoh) [inline app/index.html]: Nitpick: you could remove the second "work" here.

> ...developers who live and work in and around Philly...

**Comment by atroutt** [comment]
:shipit:

**Comment by belcherj** [comment]
Pull request Fixed. Someone want to merge

**Pull request merged**